### DISCUSSION TITLE
How to render bullet points and numbered list from markdown?

### DISCUSSION THREAD

**Discussion by elishaadamu** [sylhare/Type-on-Strap#466] [Q&A]
**Describe the bug**

**To Reproduce**

**Expected behavior**
I expected the markdown engine to render all  markdown writeups with ease, without selecting some tags

**Screenshots**
Code sample
<img width="453" alt="Screenshot 2024-08-28 230126" src="https://github.com/user-attachments/assets/ce43c174-5175-453c-9cdc-18dcd50db108">

Rendered text
<img width="290" alt="image" src="https://github.com/user-attachments/assets/b862d05a-d799-40f8-8d4e-3db3be2ced62">

**Additional context**

**Comment by sylhare** [comment]
@elishaadamu Since there's not a lot of information, I assume that the problem is coming from the formatting of the bullet points or numbered list.
With Jekyll the markdown needs to be more strictly formatted:
```
Numbered list:
1. first
2. second
3. third
  - third and a half

Bullet points:
- bullet point
- other point
    - more points
```

This will be formatted as:
![image](https://github.com/user-attachments/assets/505a902b-2d40-434d-b63b-f4933103e375)

There are some limitation but it should handle the rendering fine. It is not really a part of the theme itself as the rendering is done through the built in engine in jekyll: kramdown.

**Reply by sylhare** [reply]
@elishaadamu if that solves your issue you can click on `Mark as answer` 👍

**Comment by elishaadamu** [comment]
Just figured out the problem, adding the cdn of tailwind css, breaks the markdown rendering for some tags:     <script src="https://cdn.tailwindcss.com"></script>

**Reply by sylhare** [reply]
Ok, I'll close this discussion then.
